fastlane_core/spec/configuration_spec.rb

```diff
@@ -234,6 +235,18 @@
           expect(config[:false_value]).to eq(false)
           expect(config[:false_value2]).to eq(false)
         end
+
+        it "auto converts strings to integers" do
```

Comment:
Let's add another test for the value `"0"`, just because we mentioned it as an edge case before 👍